src/main/java/com/epam/ta/reportportal/auth/acl/ShareableObjectsHandler.java

```diff
@@ -82,9 +81,8 @@ public void initAcl(Object object, String owner, Long projectId, boolean isShare
 	public void updateAcl(Object object, Long projectId, boolean isShared) {
 		if (isShared) {
 			userRepository.findUsernamesWithProjectRolesByProjectId(projectId).forEach((key, value) -> {
-				if (ProjectRole.PROJECT_MANAGER.higherThan(value)) {
-					aclService.addPermissions(object, key, BasePermission.READ);
-				} else {
+				aclService.addPermissions(object, key, BasePermission.READ);
+				if (!ProjectRole.PROJECT_MANAGER.higherThan(value)) {
```

Comment:
let's make this condition more understandable using for example 'lowerThan'

Reply:
SameOrLower method has been added